Samples/UnifiedAPI/ArmoniK.Samples.Client/LargePayloadTests.cs

```diff
@@ -0,0 +1,176 @@
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+using ArmoniK.DevelopmentKit.Client.Exceptions;
+using ArmoniK.DevelopmentKit.Client.Factory;
+using ArmoniK.DevelopmentKit.Client.Services;
+using ArmoniK.DevelopmentKit.Client.Services.Submitter;
+
+using Microsoft.Extensions.Logging;
+
+namespace ArmoniK.Samples.Client
+{
+  internal class LargePayloadTests
+  {
+    internal static void LargePayloadSubmit(Properties     properties,
+                                            ILoggerFactory loggerFactory)
+    {
+      var logger         = loggerFactory.CreateLogger<LargePayloadTests>();
+      var sessionService = ServiceFactory.CreateService(properties);
+
+      var numbers = new List<double>
+                    {
+                      1.0,
+                      2.0,
+                      3.0,
+                      3.0,
+                      3.0,
+                      3.0,
+                      3.0,
+                      3.0,
+                    }.ToArray();
+
+      var handler = new ResultForLargeTaskHandler(logger);
+
+      var periodicInfo = ComputeVector(sessionService,
+                                       logger,
+                                       1000,
+                                       64000,
+                                       handler);
+
+      sessionService.Dispose();
+      periodicInfo.Dispose();
+    }
+
+    /// <summary>
+    ///   The first test developed to validate dependencies subTasking
+    /// </summary>
+    /// <param name="sessionService"></param>
+    /// <param name="logger"></param>
+    /// <param name="nbTasks">The number of task to submit</param>
+    /// <param name="nbElement">The number of element n x M in the vector</param>
+    /// <param name="resultForLargeTaskHandler"></param>
+    /// <param name="cancellationTokenSource"></param>
+    private static IDisposable ComputeVector(Service                    sessionService,
+                                             ILogger<LargePayloadTests> logger,
+                                             int                        nbTasks,
+                                             int                        nbElement,
+                                             ResultForLargeTaskHandler  resultForLargeTaskHandler)
+    {
+      var       indexTask        = 0;
+      var       prevIndex        = 0;
+      const int elapsed          = 30;
+      const int workloadTimeInMs = 1;
+
+      var numbers = Enumerable.Range(0,
+                                     nbElement)
+                              .Select(x => (double)x)
+                              .ToArray();
+      logger.LogInformation($"===  Running from {nbTasks} tasks with payload by task {nbElement * 8 / 1024} Ko Total : {nbTasks * nbElement / 128} Ko...   ===");
```

Comment:
```suggestion
      logger.LogInformation($"===  Running from {nbTasks} tasks with payload by task {nbElement * 8 / 1024} Ko Total : {nbTasks * nbElement * 8 / 1024} Ko...   ===");
```
For consistency

Reply:
In fact it will create bug with large number of element * nbTasks bigger than the default integer size ... i.e 1*0 000 tasks *  200 000 elements. The display will be - 16,4455 Ko/s something like that. No need a long type just divide before.
For consistency the other 8 / 1024 will be replaced